manifest/product-config.json

```
@@ -274,14 +274,24 @@
             "start_build": 1
         },
         "manifest/3.0.xml": {
-            "release": "3.0.0",
-            "release_name": "Couchbase Sync Gateway 3.0.0",
+            "release": "3.0.1",
+            "release_name": "Couchbase Sync Gateway 3.0.1",
             "production": true,
             "interval": 120,
             "go_version": "1.16.6",
             "trigger_blackduck": true,
             "start_build": 529
```

This now resets to build number 1 for 3.0.1
Please check the dev knowledgebase steps makes this clear

```suggestion
            "start_build": 1
```

---

That has been updated in the dev knowledgebase steps